dev/fattest.simplicity/src/com/ibm/websphere/simplicity/config/ServerConfiguration.java

```diff
@@ -65,6 +65,9 @@
     @XmlElement(name = "httpSession")
     private HttpSession httpSession;
 
+    @XmlElement(name = "httpSessionCache")
+    private ConfigElementList<HttpSessionCache> httpSessionCaches;
```

Comment:
Is there a reason this is a list?  My understanding was that there would only ever be one HttpSessionCache element in the server configuration, and neither httpSession nor httpSessionDatabase are lists.

Reply:
While httpSessionCache is a singleton, it is perfectly valid to configure multiple elements, and they should merge.  Without the ability to define multiple, it would be impossible to cover merging scenarios, if someone wanted to test that.  So the approach of using a list is more correct.